js/src/dapps/registry/Lookup/lookup.js

```diff
@@ -94,6 +69,7 @@ class Lookup extends Component {
             <MenuItem value='IMG' primaryText='IMG – hash of a picture in the blockchain' />
             <MenuItem value='CONTENT' primaryText='CONTENT – hash of a data in the blockchain' />
             <MenuItem value='reverse' primaryText='reverse – find a name for an address' />
+            <MenuItem value='owner' primaryText='owner – find a the owner' />
```

Comment:
Can we just double-check with @gavofyork on this - there was obviously reasons why we didn't want transfers (squatting), just want to make doubly-sure on the ability to actually display the owner of anything to anybody. (Do we want it sem-opaque o fully transparent)